doc/source/api/quantity_dimensions.rst

```diff
@@ -0,0 +1,10 @@
+.. _ref_quantity_dimensions:
+
+=====================
+Quantity dimensions
+=====================
```

Comment:
Actually this should look as follows: 

![image](https://github.com/user-attachments/assets/1067bec7-989a-43b4-954b-52c0ac49460b)

(`=` should match exactly to the length of the title) though it generates docs but gives warning for this case

If I try to reduce it's length here in GitHub suggestion then it becomes too small, not sure about why this is happening. 

Same is applicable to `Variable descriptor` title in next file.